.travis-build.sh

```diff
@@ -71,13 +75,13 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   ##  * daikon-typecheck: (takes 2 hours)
 
   # checker-framework-inference: 18 minutes
-  (cd .. && git clone --depth 1 https://github.com/typetools/checker-framework-inference.git)
+  (cd .. && git clone --depth 1 https://github.com/${SLUGOWNER}/checker-framework-inference.git)
   export AFU=`pwd`/../annotation-tools/annotation-file-utilities
   export PATH=$AFU/scripts:$PATH
   (cd ../checker-framework-inference && gradle dist && ant -f tests.xml run-tests)
 
   # plume-lib-typecheck: 30 minutes
-  (cd .. && git clone https://github.com/mernst/plume-lib.git)
+  (cd .. && git clone https://github.com/${SLUGOWNER}/plume-lib.git)
```

Comment:
But every place that use mernst/plume-lib should be changed to typetools/plume-lib?  That's a much bigger change than what you have done here.  For example, the plume-lib documentation would need to be update: https://github.com/mernst/plume-lib/blob/master/README.  Daikon would  need to be update to use typetools/plume-lib. etc.

More importantly plume-lib isn't about types, so it doesn't make sense for it to be in the typetools organization.